tests/unit/tuf_repository_service_worker/test_repository.py

```diff
@@ -396,15 +404,23 @@ def test__update_snapshot_specific_targets(self, test_repo, monkeypatch):
             "update_roles_version",
             pretend.call_recorder(lambda *a: None),
         )
+
+        def fake__bump_and_persist(md, role, **kw):
+            if role == Snapshot.type:
+                md.signed.version += 1
+
         test_repo._bump_and_persist = pretend.call_recorder(
-            lambda *a, **kw: None
+            fake__bump_and_persist
         )
         test_repo._persist = pretend.call_recorder(lambda *a: None)
 
         targets = ["bins-e"]
         result = test_repo._update_snapshot(targets)
 
-        assert result is snapshot_version
+        assert result == 4
+        assert mocked_snapshot.signed.version == 4
```

Comment:
See my other comment regarding this check.
```suggestion
        assert result == snapshot_version + 1
        assert mocked_snapshot.signed.version == snapshot_version + 1
```